R/pkg/R/mllib.R

```diff
@@ -1828,13 +1849,13 @@ setMethod("summary", signature(object = "RandomForestRegressionModel"),
 #' @note summary(RandomForestClassificationModel) since 2.1.0
 setMethod("summary", signature(object = "RandomForestClassificationModel"),
           function(object) {
-            ans <- summary.randomForest(object)
+            ans <- summary.treeEnsemble(object)
             class(ans) <- "summary.RandomForestClassificationModel"
             ans
           })
 
 #  Prints the summary of Random Forest Regression Model
-print.summary.randomForest <- function(x) {
+print.summary.treeEnsemble <- function(x) {
```

Comment:
Possibly. What would you suggest we show?
In R, generally the evaluated error should be show in summary, we don't really have that handy. Also I seems to recall an ongoing issue on the lack of consistency (or lack of information) to display to R user, and it has been suggested we should have helper functions on the model  so we could be consistent across the board (as suppose to print.summary.GeneralizedLinearRegressionModel)?

I feel like there is a lot of work we could be doing here.